routes/api/collectors.js

```diff
@@ -0,0 +1,64 @@
+var express = require('express');
+var router = express.Router();
+var gcloud = require('gcloud');
+
+var datastore = gcloud.datastore({
+  projectId: 'prototype-content-consumer',
+  keyFilename: './credentials/private-key.json'
+});
+
+/* fields => {
+               productname: chair,
+               hostname: www.ikea.com,
+               pageurl: http://www.ikea.com/us/en/catalog/products/S89100143/,
+               ...
+             }
+*/
+
+function saveCollector(fields) {
```

Comment:
Let's consider how to abstract this out into a separate models js in the codebase.